packages/react-query/bin/suspensive-react-query-switch.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+'use strict'
+import '../scripts/switch.js'
```

Comment:
In my opinion, it seems possible to change to TypeScript. Let's try this in the next version.